prep-css-grid-classes/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Grid Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+/* stylelint-disable block-no-empty */
+body {
+  font-family: sans-serif;
+}
+
+h1 {
+  text-align: center;
+}
+
+.container {
+  max-width: 992px;
+  margin: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+}
+
+.column {
+  flex-basis: calc(100% / 3);
+  padding-left: 8px;
+  padding-right: 8px;
+}
+
+  </style>
+</head>
+<body>
+
+<h1>Indiana Jones</h1>
+
+<div class="container">
+<div class="row">
+<div class="column">
+  <img style:width;100% src="images/crusade.png" alt="The Last Crusade" >
```

Comment:
It's better to  put your styling in the CSS code instead of using a `style` attribute.